docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -177,14 +177,14 @@ Here is a complete list of options for Gradle tasks:
 
 ### Attributes common to JVM and JS
 
-| Name | Description | Possible values |Default value |
-|------|-------------|-----------------|--------------|
-| `allWarningsAsErrors` | Report an error if there are any warnings |  | false |
-| `suppressWarnings` | Don't generate warnings |  | false |
-| `verbose` | Enable verbose logging output. Works only when the [Gradle debug log level enabled](https://docs.gradle.org/current/userguide/logging.html) |  | false |
-| `freeCompilerArgs` | A list of additional compiler arguments. You can use experimental `-X` arguments here too. See an [example](#example-of-additional-arguments-usage-via-freecompilerargs) |  | [] |
-| `apiVersion`      | Restrict the use of declarations to those from the specified version of bundled libraries | "1.6", "1.7", "1.8", "1.9", "2.0", "2.1" (EXPERIMENTAL) |               |
-| `languageVersion` | Provide source compatibility with the specified version of Kotlin                         | "1.6", "1.7", "1.8", "1.9", "2.0", "2.1" (EXPERIMENTAL) |               |
+| Name | Description | Possible values                                                |Default value |
+|------|-------------|----------------------------------------------------------------|--------------|
+| `allWarningsAsErrors` | Report an error if there are any warnings |                                                                | false |
+| `suppressWarnings` | Don't generate warnings |                                                                | false |
```

Comment:
Kotlin `2.1.0` release has new stable `extraWarnings` compiler option and I think it should be mentioned here as well. This option is common for all targets.

Reply:
This will be covered in a separate PR by @danil-pavlov as he is covering this feature :)